src/Bridges/Objective/map.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    Map <: AbstractDict{MOI.ObjectiveFunction, AbstractBridge}
+
+Mapping between bridged variables and the bridge that bridged the variable.
```

Comment:
I don't think this is the correct sentence. Maybe:

A mapping from a bridged objective function to the bridge responsible for creating that objective function.